src/journal/journald-server.c

```diff
@@ -873,14 +945,14 @@ static void write_to_journal(Server *s, uid_t uid, struct iovec *iovec, size_t n
         }
 
         if (vacuumed || !shall_try_append_again(f->file, r)) {
-                log_error_errno(r, "Failed to write entry (%zu items, %zu bytes), ignoring: %m", n, IOVEC_TOTAL_SIZE(iovec, n));
+                log_failed_entry(LOG_ERR, r, NULL, ", ignoring", n, IOVEC_TOTAL_SIZE(iovec, n));
```

Comment:
We can pass `f->file->path` here as well I think, it was probably just an oversight that we didn't include it in the log message before